awx/main/management/commands/register_default_execution_environments.py

```diff
@@ -84,21 +84,28 @@ def handle(self, *args, **options):
 
             registry_cred.inputs = inputs
             registry_cred.save()
-            changed = True
-
-            if not created:
-                print('Updated Default Execution Environment Credential')
-
-        for ee in reversed(settings.DEFAULT_EXECUTION_ENVIRONMENTS):
-            _, created = ExecutionEnvironment.objects.update_or_create(
-                name=ee["name"], defaults={"image": ee["image"], "managed_by_tower": True, "credential": registry_cred}
-            )
-
-        if created:
-            changed = True
-            print("Default Execution Environment(s) registered.")
+            changed.append(True)
+
+            if not cred_created:
+                print("Updated 'Default Execution Environment Credential'")
+
+        # Create default globally available Execution Environments
+        for ee in reversed(settings.GLOBAL_JOB_EXECUTION_ENVIRONMENTS):
+            _, ee_created = ExecutionEnvironment.objects.update_or_create(name=ee["name"], image=ee["image"], credential=registry_cred)
```

Comment:
so here, we lose the `managed_by_tower` flag for the list of EEs from the pre-creation list in settings.

but let's keep in mind that this is how the default EE gets grabbed:

https://github.com/ansible/awx/blob/68f214c2bec56af99129c3c8971cb7eaab8bd9ca/awx/main/utils/execution_environments.py#L9-L12

I thought we said that the list would grow some means of declaring 1 out of the list as the job default (`DEFAULT_EXECUTION_ENVIRONMENT`). Maybe that's still a TODO for @jbradberry

Reply:
To copy over info discussed in #10378, the most recently created EE is the one that gets prioritized.  Keep that in mind when specifying the order of EEs in the variable.